src/packages/block/manifests.ts

```diff
@@ -4,8 +4,11 @@ import { manifests as blockListManifests } from './block-list/manifests.js';
 import { manifests as blockRteManifests } from './block-rte/manifests.js';
 import { manifests as blockTypeManifests } from './block-type/manifests.js';
 import type { ManifestTypes } from '@umbraco-cms/backoffice/extension-registry';
+// TODO: Remove test custom view, or transfer to test or similar?
```

Comment:
So we have overcome this before by adding custom manifests to the manifest MSW handler, which you can then turn on by setting the `VITE_UMBRACO_EXTENSION_MOCKS` to `on`. I would recommend you move this code to that handler. That way you can always turn it "on" by a simple environment value.